### PR TITLE
Add temperature conversion to Int

### PR DESCRIPTION
**Pull request by ChristopherAaronBrown**
:rocket: Added C -> F and F -> C for Ints

### CODE REVIEW

**Comment by LucianoPAlmeida** [comment]
Hey @ChristopherAaronBrown Thanks to taking the time for that PR 👍 
But I think it is the same case of #387 where I think those are not a good fit for Int/Double extensions because of the following reasons:

* Not every Int represents a measure. So assuming that Int represents a temperature measure makes the API not concise. For example:
let distance: Int = 20
distance.fahrenheitToCelsius 
In the example above we can see that feels strange a Int in one level of abstraction that represents a distance have a property that converts it to celsius.

* Double is a trivial type that it's only responsibility is to represent a number and because of separation of concerns in API's it makes more sense separate measure or any other responsibility in other layers of abstraction :)

* Foundation already has a really great and simple API to deal with measurement. Example:

let measure = Measurement<UnitTemperature>(value: 20, unit: UnitTemperature.celsius) 
measure.converted(to: UnitLength.fahrenheit) 

TLDR; The reason is just to maintain the API concise and correct 
Again thank you for taking your time to submit this PR :))

**Comment by ChristopherAaronBrown** [comment]
I see, that makes sense. Thanks for reviewing my pull request!
